test/state/host.cpp

```diff
@@ -0,0 +1,368 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2022 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "host.hpp"
+#include "rlp.hpp"
+
+namespace evmone::state
+{
+bool Host::account_exists(const address& addr) const noexcept
+{
+    const auto* const acc = m_state.get_or_null(addr);
+    return acc != nullptr && (m_rev < EVMC_SPURIOUS_DRAGON || !acc->is_empty());
+}
+
+bytes32 Host::get_storage(const address& addr, const bytes32& key) const noexcept
+{
+    const auto& acc = m_state.get(addr);
```

Comment:
Wouldn't this put it into the hot list?

Reply:
This assumes the account must exist already. And it is hot already.